### PR TITLE
[T2A5][T16-A3] Xu Tianyu

### PR DESCRIPTION
**Pull request by yeetee179**
Find with lower case

### CODE REVIEW

**Comment by mauris** [comment]
@yeetee179 do note there is no space around the dash in your team ID for future submissions.

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -217,6 +212,16 @@ public static void main(String[] args) {
      * signature anyway.
      * ====================================================================
      */
+	 
+	 public static void promptUserInput(){
+		 while (true) {
```
Comment (lohvht-zz) [inline src/seedu/addressbook/AddressBook.java:217]: Take note of the indentations here, the while(true) line is a tab space more than the rest of the code

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -420,7 +425,9 @@ private static String getMessageForSuccessfulAddPerson(String[] addedPerson) {
      * @return feedback display message for the operation result
      */
     private static String executeFindPersons(String commandArgs) {
+		commandArgs = commandArgs.toLowerCase();
```
Comment (lohvht-zz) [inline src/seedu/addressbook/AddressBook.java:428]: indentation here too

**Comment by lohvht-zz** [comment]
Hi, good try on implementing the find lowercase, Do take note of the proper indentations, especially when you're copying and pasting code, alternatively, if you want to extract the method you can use Eclipse's refactoring to extract the method, that way you do not need to worry about getting the wrong indentation level.

Also take note that whenever you implement a new feature, always modify the header comment, and update tests and user guides.

**Pull request closed without merging**